omod/src/main/webapp/pages/htmlform/editHtmlFormWithSimpleUi.gsp

```diff
@@ -29,7 +29,7 @@ ${ ui.includeFragment("coreapps", "patientHeader", [ patient: patient ]) }
     var breadcrumbs = _.flatten([
         { icon: "icon-home", link: '/' + OPENMRS_CONTEXT_PATH + '/index.htm' },
         ${ breadcrumbMiddle },
-        { label: "${ ui.escapeJs(ui.message("emr.editHtmlForm.breadcrumb", ui.format(htmlForm.form))) }" }
+        { label: "${ ui.escapeJs(ui.format(htmlForm.form)) }" }
```

Comment:
This seems like it is changing the previously-intended behavior.
You can see what that message is supposed to be in different languages here: https://github.com/PIH/openmrs-module-emr/search?utf8=%E2%9C%93&q=emr.editHtmlForm.breadcrumb